README.md

```diff
@@ -16,21 +16,25 @@ Cubicasa SDK makes possible you to add scanning view to your app which then can
 ## Release Notes
 
 - Depth capturing support for the following devices; iPhone 12 Pro, iPhone 12 Pro Max, iPad Pro 2020
-- Depth visualization to show which spaces have been scanned (visualization colours can be changed)
+- Mesh visualisation to show which spaces have been scanned (visualisation colours can be changed, visualisation can be turned off altogether)
 - New status codes (code 3, 5, 18, 27, 29, 30 and 40-49 and 57 to match Android SDK statuses)
 - Updated dependencies
 - Speech recognition for room labels (Now you can directly say the labels you want to have in the finished floorplan, UI colours can be changed)
 - Adjusted lighting for dark spaces (Camera flash will turn on in dark spaces to ensure tracking and data quality)
 - UI adjustments for new features see #Updating from CubiCapture 2.0 to CubiCapture 2.2
+- Debug symbols (`dSYM`) included in the `CubiCapture.xcframework`
+- Various bugfixes
 
```

Comment:
Add viewcontroller change to release notes and direct them how to change the implementation

Reply:
Done.